<ui version="4.0" >
 <class>qdOptions</class>
 <widget class="QDialog" name="qdOptions" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Options</string>
  </property>
  <widget class="QDialogButtonBox" name="qdbbRespond" >
   <property name="geometry" >
    <rect>
     <x>440</x>
     <y>220</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QStackedWidget" name="qswChoices" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>0</y>
      <width>441</width>
      <height>201</height>
     </rect>
    </property>
    <property name="currentIndex" >
     <number>5</number>
    </property>
    <widget class="QWidget" name="page_4" />
    <widget class="QWidget" name="page_5" >
     <widget class="QGroupBox" name="groupBox" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>10</y>
        <width>120</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title" >
       <string>Files</string>
      </property>
      <widget class="QCheckBox" name="qcbShowSystemFiles" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>111</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text" >
        <string>Show system files</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="qcbShowHiddenFiles" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>40</y>
         <width>111</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text" >
        <string>Show hidden files</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5" >
      <property name="geometry" >
       <rect>
        <x>140</x>
        <y>10</y>
        <width>120</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title" >
       <string>Files size in</string>
      </property>
      <widget class="QRadioButton" name="qrbSizeBytes" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>83</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text" >
        <string>Bytes</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="qrbSizeKilobytes" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>40</y>
         <width>83</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text" >
        <string>Kilobytes</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="qrbSizeMegabytes" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>60</y>
         <width>83</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text" >
        <string>Megabytes</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="qrbSizeDynamic" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>80</y>
         <width>83</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text" >
        <string>Dynamic</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3" >
     <widget class="QComboBox" name="qcbColumnSet" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>30</y>
        <width>67</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="qpbColumnSetAdd" >
      <property name="geometry" >
       <rect>
        <x>80</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qpbColumnSetRemove" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>160</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Remove</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qpbColumnUp" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>420</x>
        <y>80</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qpbColumnDown" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>420</x>
        <y>110</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="label" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>10</y>
        <width>57</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text" >
       <string>&amp;Column set:</string>
      </property>
      <property name="buddy" >
       <cstring>qcbColumnSet</cstring>
      </property>
     </widget>
     <widget class="QTreeWidget" name="qtwColumns" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>60</y>
        <width>411</width>
        <height>101</height>
       </rect>
      </property>
      <property name="rootIsDecorated" >
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="qpbColumnAdd" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>170</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qpbColumnRemove" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>80</x>
        <y>170</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Remove</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2" >
     <widget class="QGroupBox" name="groupBox_4" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>10</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title" >
       <string>Time display</string>
      </property>
      <widget class="QLabel" name="label_3" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>40</y>
         <width>65</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text" >
        <string>%s - seconds</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text" >
        <string>%h - hours, %m - minutes,</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="qlePluginTimeDisplay" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>60</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page" >
     <widget class="QTreeWidget" name="qtwContentPlugins" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>10</y>
        <width>441</width>
        <height>151</height>
       </rect>
      </property>
      <property name="rootIsDecorated" >
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="qpbAddContentPlugin" >
      <property name="geometry" >
       <rect>
        <x>280</x>
        <y>170</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qpbRemoveContentPlugin" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>360</x>
        <y>170</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Remove</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8" >
     <widget class="QPushButton" name="qpbRemoveListerPlugin" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>360</x>
        <y>170</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Remove</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qpbAddListerPlugin" >
      <property name="geometry" >
       <rect>
        <x>280</x>
        <y>170</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Add</string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="qtwListerPlugins" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>10</y>
        <width>441</width>
        <height>151</height>
       </rect>
      </property>
      <property name="rootIsDecorated" >
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6" />
    <widget class="QWidget" name="page_7" >
     <widget class="QGroupBox" name="groupBox_2" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title" >
       <string>File overwrite</string>
      </property>
      <widget class="QRadioButton" name="qrbOverwriteAll" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>40</y>
         <width>83</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text" >
        <string>Overwrite all</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="qrbOverwriteAllOlder" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>60</y>
         <width>111</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text" >
        <string>Overwrite all older</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="qrbOverwriteSkipAll" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>80</y>
         <width>83</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text" >
        <string>Skip all</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="qrbOverwriteAsk" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>83</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text" >
        <string>Ask</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3" >
      <property name="geometry" >
       <rect>
        <x>150</x>
        <y>10</y>
        <width>130</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title" >
       <string>Readonly file overwrite</string>
      </property>
      <widget class="QRadioButton" name="qrbReadonlyOverwriteAsk" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>83</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text" >
        <string>Ask</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="qrbReadonlyOverwriteYesToAll" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>40</y>
         <width>83</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text" >
        <string>Yes to all</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="qrbReadonlyOverwriteNoToAll" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>60</y>
         <width>83</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text" >
        <string>No to all</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QTreeWidget" name="qtwChoices" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>120</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
